src/misc.cc

```diff
@@ -99,13 +99,16 @@ namespace
 // - after negative answer, call solve_and_commit() again
 bool confirm_licenses( Zypper & zypper )
 {
-  bool confirmed = true;
+  std::map<std::string,std::set<PoolItem>> licenseCollector; // bsc#1145554: collect multiple packages using the same license text
```

Comment:
we could use a unordered_map , or do we care about preserving the order?

Reply:
No, we could use an unordered_map as well (there is no order to preserve).